README.md

```diff
@@ -47,8 +50,8 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with
    available to purchase.
 
    - Go to the `Settings` tab, then to `API Keys`.
-   - Create a new API Key, name it whatever you like, `starter` would be fine.
-   - Give it the `anonymous` role.
+   - Create a new API Key and name it whatever you like, such as `starter`.
+   - Give it `read` permissions.
```

Comment:
Check out the kitchen sink starter's roles. The roles should be given clear names that refer to the user and their status and capabilities, rather than being named for the permissions they might confer. For this project you should have two roles, `authenticated` and `anonymous`.